package.json

```diff
@@ -21,7 +21,7 @@
     "record-build-info": "node ./bin/record-build-info",
     "lint": "eslint . --cache --max-warnings 0",
     "typecheck": "tsc",
-    "test": "jest --collectCoverage=true --testPathPattern=/server/.*",
+    "test": "jest --collectCoverage=true --detectOpenHandles",
```

Comment:
I think detectOpenHandles has a bit of a performance cost

Reply:
remove it?